gulpfile.js

```diff
@@ -216,9 +216,7 @@ function librarySizeTask() {
 
 function moduleSizesTask() {
   return gulp.src(srcDir + '**/*.js')
-    .pipe(uglify({
-      preserveComments: 'some'
-    }))
+    .pipe(uglify())
```

Comment:
`preserveComments` has been removed in v3, but we still need to set the new option, right?